File: 231-power-of-two/231-power-of-two.java
class Solution {
    public boolean isPowerOfTwo(int n) {
        if(n == 0) return false;
        
        while(true){
            if(n == 1) return true;
            if(n % 2 != 0) return false;
            n = n / 2;
        }
    }
}

File: 231-power-of-two/231-power-of-two.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfTwo = function(n) {
    if(n === 0) return false;
    while(true){
        if(n === 1) return true;
        if(n % 2 !== 0) return false;
        n = n / 2;
    }
};